README.md

```diff
@@ -191,7 +248,6 @@ cf autoscaling-metrics APP_NAME METRIC_NAME [--start START_TIME] [--end END_TIME
 
 
 #### OPTIONS:
-- `METRIC_NAME` : available metric supported: memoryused, memoryutil, responsetime, throughput and cpu.
```

Comment:
- `METRIC_NAME` :  default metrics "memoryused, memoryutil, responsetime, throughput, cpu"  and customized name for your own metrics